repo/packages/N/nifi/500/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "packagingVersion": "4.0",
+  "upgradesFrom": [
+    "0.2.0-1.5.0"
```

Comment:
hi @Gupta-Amrit we should just leave upgradesFrom and downgradesTo empty for now. Once the nifi becomes certified we need to set this proper and also test it.

cc @ryadav88